packages/postgres-database/src/simcore_postgres_database/models/funcapi_function_job_collections_table.py

```diff
@@ -0,0 +1,51 @@
+"""Functions table
+
+- List of functions served by the simcore platform
+"""
+
+import uuid
+
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.sql import func
+
+from .base import metadata
+
+function_job_collections_table = sa.Table(
```

Comment:
I like the suffix `_table`. It is not used in the other tables but i can think of situations in which i would have liked to have that in the name.